packages/yoastseo/spec/fullTextTests/testTexts/en/englishPaper.html

```diff
@@ -30,7 +30,7 @@ <h2>Voice results do not (always) make sense</h2>
     As a reader,  you can scan a text, you can skip pieces of information or read an important paragraph twice.
     You cannot do that as a listener. As a reader, you&#8217;re much more in control.
     So, if you&#8217;re searching for information about what to do in Barcelona, it makes much more sense to get that information from a book or a screen.</p>
-<h2>Search engines are growing towards singular results</h2>
+<h2 class="has-text-align-center">Search engines are growing towards singular results</h2>
```

Comment:
I edited `englishPaper.html` so that the heading is center-aligned. This way it can trigger and give feedback for Text Alignment assessment